tools/syncer.md

```diff
@@ -490,35 +504,40 @@ Syncer provides the metric interface, and requires Prometheus to actively obtain
 
 #### title: binlog events
```

Comment:
@lilin90 @csuzhangxc any problem with these metric descriptions, if you don't have any problems, I would also fill them in syncer monitor template

Reply:
no problem.